packages/nx/src/project-graph/utils/retrieve-workspace-files.ts

```diff
@@ -78,11 +78,37 @@ export async function retrieveWorkspaceFiles(
  */
 export async function retrieveProjectConfigurations(
   workspaceRoot: string,
-  nxJson: NxJsonConfiguration
+  nxJson: NxJsonConfiguration,
+  includeProjectsFromAngularJson: boolean = false
```

Comment:
Can we create a separate function that has this as `true`?

`retrieveProjectConfigurationsWithAngularProjects`

Reply:
added